src/inverseRobot.js

```diff
@@ -7,7 +7,31 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const inversedRobot = {};
+  const arrKeys = [];
+  const arrValues = [];
+
+  for (const key in robot) {
+    arrKeys.push(key);
+  }
+
+  for (const key in robot) {
+    arrValues.push(robot[key]);
+  }
+
+  for (let i = 0; i < arrValues.length; i++) {
+    for (let j = i + 1; j < arrValues.length; j++) {
+      if (arrValues[i] === arrValues[j]) {
+        return null;
+      }
+    }
+  }
+
+  for (let i = 0; i < arrKeys.length; i++) {
+    inversedRobot[arrValues[i]] = arrKeys[i];
+  }
```

Comment:
do you really need 5 loops, with two of them nested? Actually this task can be solved with just one loop